src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/MinioIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2018 CloudBees, Inc.
```

Comment:
2020?

And do we really need a license statement at the beginning of a test class?

Reply:
> 2020?

Came from the upstream PR, presumably copy-paste. No one enforces copyright dates that I know of.

> do we really need a license statement at the beginning of a test class?

Why not? `src/test/java/` is at as important as `src/main/java/`.

Reply:
I guess I would think we'd have a license file somewhere and not need to put it in every class.

Reply:
That would be nice, but I am afraid we put a license header at the top of every source file for all OSS repos.